scripts/new-chain.ts

```diff
@@ -0,0 +1,234 @@
+import fs from 'fs'
+import path from 'path'
+import { ethers } from 'ethers'
+import dotenv from "dotenv";
+import { runScript } from './utils';
+import { ADDRESS, CODEHASH, FACTORY_BYTECODE, SIGNER } from './constants';
+
+dotenv.config()
+
+async function newChainWrapper() {
+	let summary: { commentOutput: string, labelOperation: string } = {
+		commentOutput: "An unexpected error occurred",
+		labelOperation: "--remove-label"
+	};
```

Comment:
Question - why can't you just:

```suggestion
	let summary: { commentOutput: string, labelOperation: string };
```

Reply:
Without assigning, it results in `Variable 'summary' is used before being assigned.` in the `finally` block.